example/src/layouts/Editor.vue

```diff
@@ -215,26 +220,29 @@ export default class Editor extends Vue {
 
   private setStrokeProperties(attr: StrokeAttributes): void {
     let value;
+    let currentStrokeProps = this.strokeProps;
```

Comment:
This should be a deep copy. Should this be done in drawify instead though? Seems like an easy mistake to make.

Reply:
Ahh, I see what you're saying. Yes, I think there should be a deep copy performed in Dispatcher in drawify